### ISSUE TITLE
Way to detect if signature is empty

### ISSUE THREAD

**Issue by DevNamedZed** [RepairShopr/react-native-signature-capture#105]
Need to enforce a signature in my app. Doesnt seem like there is a way to test for an empty signature? 

I tried to use the ondrag event but that has some type of threshold that has to be crossed before it can fire.

**Comment by DevNamedZed** [comment]
The ios version does not raise an event when the signature is empty. Updated the android version to have the same behavior. 

It would be better to always raise the event, maybe with null or something to indicate empty signature.

**Comment by florismettey** [comment]
I agree with @DevNamedZed, iOS and Android don't have the same behavior and it would be better to always raise the event

**Comment by CarolynWebster** [comment]
I imagine this is too late to be helpful, but I used state within the component using the sig capture and added unchanged: true, and onDrag - unchanged changes to false, and on reset back to true.

**Comment by DevNamedZed** [comment]
I submitted a pull request to fix shortly after I posted this issue, but it does not seem like this project is really maintained

https://github.com/RepairShopr/react-native-signature-capture/pull/106/commits/6ed42b7fba3600700b0e5a719d58db3d092bd7e5

**Comment by wilav-dev** [comment]
Any updates? I need to detect if is empty
